Dockerfile

```diff
@@ -28,7 +28,7 @@ COPY requirements.in dev-requirements.in "${CWD}/"
 RUN set -eux \
     && ln -sf /usr/local/bin/python /bin/python \
     && apt-get update \
-    && apt-get install --yes --only-upgrade openssl ca-certificates \
+    && apt-get install --yes --only-upgrade openssl libssl-dev ca-certificates \
```

Comment:
Probably need to remove the `--only-upgrade` flag so it is actually installed.

Reply:
Nice catch, thanks!